### ISSUE TITLE
sqrt method

### ISSUE THREAD

**Issue by ZettaScript** [alkis/decimal#56]
A `d128.sqrt()` (like `f64.sqrt()`) method would be useful.

**Comment by tari** [comment]
I agree it would be nice to have a shortcut, though `d128::pow(0.5)` should do the job.
